UI/cmake/feature-youtube.cmake

```diff
@@ -11,3 +11,10 @@ else()
   set(YOUTUBE_SECRET_HASH 0)
   set(YOUTUBE_CLIENTID_HASH 0)
 endif()
+
+if(YOUTUBE_WEBAPP_ADDRESS && YOUTUBE_ENABLED)
```

Comment:
`YOUTUBE_ENABLED` is not a global bool. It is a private target compile definition for obs-studio. It cannot be used this way.

Reply:
Hmmmmmmm.
I moved webapp-related block inside youtube-related. Is it ok?

Please wait a minute, I need to check building with and without YOUTUBE-related staff...

Reply:
Retested, done.

Reply:
This is now resolved.